src/partition/ReceivedBoundingBox.cpp

```diff
@@ -92,8 +93,8 @@ void ReceivedBoundingBox::computeBoundingBox()
     // master receives feedbacks from slaves and put them into the feedbackmap
     for (int rank_slave=1; rank_slave < utils::MasterSlave::_size ; rank_slave++)
     {
-      utils::MasterSlave::_communication->receive(_feedback, rank_slave);
-      _feedbackMap[rank_slave]=_feedback;        
+      utils::MasterSlave::_communication->receive(_feedback, rank_slave);         
+      _feedbackMap[rank_slave]=_feedback;
```

Comment:
Why are `_feedback` and `_feedbackMap` member variables?

Reply:
not needed! changed!